From f6eaf270c1cfeaefd2a09ca084226e23e56b5793 Mon Sep 17 00:00:00 2001
From: Anson Huang <b20788@freescale.com>
Date: Tue, 10 Sep 2013 10:31:45 -0400
Subject: [PATCH 415/442] ENGR00278883 Remove rtc_timesetnotification_test for
 kernel v3.10

Kernel v3.10 do NOT support RTC_READ_TIME_47BIT and RTC_WAIT_TIME_SET
IOCTLs, so remove the unit test which use them.

Signed-off-by: Anson Huang <b20788@freescale.com>
---
 test/mxc_rtc/Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/mxc_rtc/Makefile b/test/mxc_rtc/Makefile
index 12eea11..e3bcdb5 100644
--- a/test/mxc_rtc/Makefile
+++ b/test/mxc_rtc/Makefile
@@ -2,7 +2,7 @@
 EXCLUDE_LIST:= IMX25_3STACK
 
 ifeq (,$(findstring $(PLATFORM), $(EXCLUDE_LIST)))
-OBJS = $(OBJDIR)/rtc_timesetnotification_test.out $(OBJDIR)/rtctest.out $(OBJDIR)/rtcwakeup.out $(OBJDIR)/autorun-rtc.sh
+OBJS = $(OBJDIR)/rtctest.out $(OBJDIR)/rtcwakeup.out $(OBJDIR)/autorun-rtc.sh
 else
 OBJS =
 endif
-- 
1.8.0

